packer-scripts/run-serverspecs

```diff
@@ -48,8 +48,8 @@ __install_chefdk() {
 
   apt-key add "${tmp}/chef.asc"
 
-  echo "deb https://packages.chef.io/current-apt $(lsb_release -cs) main" \
```

Comment:
ah. is this a name change for chef from `current` to `stable`  upstream fun times.